docs/components/layout/index.html

```diff
@@ -1,5 +1,5 @@
 ---
-title: Layouts
+title: Layout
```

Comment:
Our singular naming convention breaks down a little here, though not a showstopper. 👍